.github/workflows/build_linux_wheel/action.yml

```diff
@@ -20,23 +21,37 @@ runs:
       shell: bash
       run: |
         echo "ARM BUILD: ${{ inputs.arm-build }}"
-    - name: Build manylinux image
+    - name: Build x86_64 Manylinux wheel
       if: ${{ inputs.arm-build == 'false' }}
-      shell: bash
-      working-directory: python
-      run: |
-        docker build -t pylance_manylinux -f tools/Dockerfile.manylinux2014  tools/
-    - name: Build manylinux image (arm)
-      if: ${{ inputs.arm-build == 'true' }}
-      shell: bash
-      working-directory: python
-      run: |
-        docker build -t pylance_manylinux -f tools/Dockerfile.manylinux2014_aarch64 tools/
-    - name: Build wheel
       uses: PyO3/maturin-action@v1
       with:
         command: build
+        working-directory: python
+        target: x86_64-unknown-linux-gnu
+        manylinux: "2_17"
```

Comment:
Last time we bump manylinux from 2014 to something news broke some users.  @chebbyChefNEQ do you remember the details.

Reply:
2014 is just an alias for 2_17: https://peps.python.org/pep-0600/#legacy-manylinux-tags

Reply:
I think Rob was trying to upgrade to 2_28.